user-preferences-explainer.md

```diff
@@ -0,0 +1,68 @@
+# User Preferences Explainer
+
+Authors: [Aaron Gustafson](https://github.com/aarongustafson), Louise Brett (loubrett@google.com)
+
+## Overview
+
+Currently in the web app manifest a `theme_color` and `background_color` can be defined. However there is no way to change these based on dark mode or [other user preferences](#possible-extensions).
+
+This document proposes a new `user_preferences` manifest member that enables web apps to provide alternate values for manifest members given specific user preferences. The structure of this `user_preferences` field matches the structure of the proposed <code>[translations](https://github.com/WICG/manifest-incubations/blob/gh-pages/translations-explainer.md)</code> member.
+
+This new member is inspired by the [CSS media queries user preferences](https://drafts.csswg.org/mediaqueries-5/#mf-user-preferences). The keys are simple strings but they are derived from the CSS media query syntax and can easily be converted back if necessary.
```

Comment:
Converting back sets the expectation that things will then function as in CSS, which is what the issue discussion made clear would _not_ happen (see [this section](https://github.com/w3c/manifest/issues/975#:~:text=concerned%20about%20this.-,I%20don't%20think%20it's,go%20down%20this%20route.,-Contrary%20to%20what) specifically).

Reply:
Thanks for linking that, I had been looking at some older parts of the thread. I've removed that bit now.